rest/data/admin.go

```diff
@@ -172,6 +172,22 @@ func (ac *DBAdminConnector) RestartFailedTasks(queue amboy.Queue, opts model.Res
 	}, nil
 }
 
+func (ac *DBAdminConnector) RestartFailedCommitQueueVersions(opts model.RestartTaskOptions) (int, error) {
+	pRefs, err := model.FindProjectRefsWithCommitQueueEnabled()
+	if err != nil {
```

Comment:
this will return an error if there are no matching projects.
Is that what you intended?

Reply:
I believe this does not return an error in this case. Added a case to admin_test.go.

Reply:
I understand now: It seems that if the query ends up going through FindAll in db_utils.go it will not return an error while if it goes through FindOne it will.
Since here we're getting all the matching projectRefs it will not return an error, as you said.